typescript/codepipeline-using-cross-account-repos/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "codepipeline-using-cross-account-repos",
+  "version": "1.0.0",
+  "description": "a minimal codepipeline referencing a cross account codecommit repo",
+  "private": true,
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc -w",
+    "cdk": "cdk"
+  },
+  "author": {
+    "name": "Amazon Web Services",
+    "url": "https://aws.amazon.com",
+    "organization": true
+  },
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@types/node": "^8.10.38",
+    "typescript": "^3.2.4"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-kms": "*",
```

Comment:
Why “*” and not a version number? When CDK V2 is released, this example will almost certainly break.

Reply:
Thank you for all the comments, I'll work to address them

For this particular comment, I did this to be consistent with the other examples

I'm pretty sure there's even a unit test verifying that all examples use * and I had to remove my version number to make the PR green. Unfortunately I'm not sure how to find previous travis CI build failures for a PR